blips/blip-0001.mediawiki

```diff
@@ -0,0 +1,236 @@
+* Merge the corresponding pull request
+* Send a message back to the bLIP author with the next steps.
+
+The bLIP editors are intended to fulfill administrative and editorial responsibilities. They do not pass judgement on bLIPs. The bLIP editors monitor bLIP changes, and update bLIP headers as appropriate.
+
+==What belongs in a successful bLIP?==
+
+bLIPs should be written in mediawiki format.
+
+Each bLIP should have the following parts:
+
+* Preamble -- Headers containing metadata about the bLIP (see below).
+* Abstract -- A short (~200 word) description of the technical issue being addressed.
+* Copyright -- The bLIP must be explicitly licensed under acceptable copyright terms (see below).
+* Specification -- The technical specification should describe the syntax and semantics of any new feature. The specification should be detailed enough to allow competing, interoperable implementations for any of the current Lightning implementations.
+* Motivation -- The motivation is critical for bLIPs that want to change the Lightning protocol. It should clearly explain why the existing protocol is inadequate to address the problem that the bLIP solves.
+* Rationale -- The rationale fleshes out the specification by describing what motivated the design and why particular design decisions were made. It should describe alternate designs that were considered and related work. The rationale should provide evidence of consensus within the community and discuss important objections or concerns raised during discussion.
+* Backwards compatibility -- All bLIPs that introduce backwards incompatibilities must include a section describing these incompatibilities and their severity. The bLIP must explain how the author proposes to deal with these incompatibilities.
+* Reference implementation -- The reference implementation must be completed before any bLIP is given status "Final", but it need not be completed before the bLIP is accepted. It is better to finish the specification and rationale first and reach consensus on it before writing code. The final implementation must include test code and documentation appropriate for the Lightning protocol.
+
+====bLIP Header Preamble====
+
+Each bLIP must begin with an RFC 822 style header preamble. The headers must appear in the following order. Headers marked with "*" are optional and are described below. All other headers are required.
+
+<pre>
+bLIP: bLIP number, this is determined by the bLIP editor
+Title: bLIP title
+Author: list of the author's or authors' name(s) and/or username(s), or name(s) and email(s). Details are below.
+* Discussions-To: a url pointing to the official discussions thread
+Status: Draft, Active, Proposed, Deferred, Rejected, Withdrawn, Final, Replaced, Obsolete
+Type: Standards Track, Informational, Process
+Category: Core, Networking, Routing, Interface, or LRC (fill out for Standards Track bLIPs only)
```

Comment:
As stated above, I think we should start without these and see what emerges.
We can add more structure later, but constraining too early is not a good idea IMHO.